Index: kde4support-4.95.0/src/CMakeLists.txt
===================================================================
--- kde4support-4.95.0.orig/src/CMakeLists.txt	2014-01-04 14:38:11.000000000 +0000
+++ kde4support-4.95.0/src/CMakeLists.txt	2014-01-30 14:16:40.667852508 +0000
@@ -518,8 +518,8 @@
 install( FILES kdebug.areas kdebugrc DESTINATION ${CONFIG_INSTALL_DIR} )
 
 configure_file(kdecore/all_languages.desktop ${CMAKE_CURRENT_BINARY_DIR}/all_languages @ONLY)
-install( FILES ${CMAKE_CURRENT_BINARY_DIR}/all_languages DESTINATION ${LOCALE_INSTALL_DIR}  )
-install( FILES kdecore/entry.desktop  DESTINATION  ${LOCALE_INSTALL_DIR}/en_US )
+install( FILES ${CMAKE_CURRENT_BINARY_DIR}/all_languages DESTINATION ${LOCALE_INSTALL_DIR} RENAME kf5_all_languages )
+install( FILES kdecore/entry.desktop  DESTINATION  ${LOCALE_INSTALL_DIR}/en_US RENAME kf5_entry.desktop )
 
 
 install( FILES kdeui/40.colors kdeui/Oxygen.colors kdeui/Web.colors kdeui/Royal.colors kdeui/Rainbow.colors
Index: kde4support-4.95.0/src/kdecore/klocale_kde.cpp
===================================================================
--- kde4support-4.95.0.orig/src/kdecore/klocale_kde.cpp	2014-01-04 14:38:11.000000000 +0000
+++ kde4support-4.95.0/src/kdecore/klocale_kde.cpp	2014-01-30 14:07:53.319857537 +0000
@@ -2695,7 +2695,7 @@
 QStringList KLocalePrivate::allLanguagesList()
 {
     if (!m_languages) {
-        m_languages = new KConfig(QLatin1String("locale/all_languages"), KConfig::NoGlobals, QStandardPaths::GenericDataLocation);
+        m_languages = new KConfig(QLatin1String("locale/kf5_all_languages"), KConfig::NoGlobals, QStandardPaths::GenericDataLocation);
     }
     return m_languages->groupList();
 }
@@ -2720,7 +2720,7 @@
 QString KLocalePrivate::languageCodeToName(const QString &language)
 {
     if (!m_languages) {
-        m_languages = new KConfig(QLatin1String("locale/all_languages"), KConfig::NoGlobals, QStandardPaths::GenericDataLocation);
+        m_languages = new KConfig(QLatin1String("locale/kf5_all_languages"), KConfig::NoGlobals, QStandardPaths::GenericDataLocation);
     }
 
     KConfigGroup cg(m_languages, language);
